views/conditions/head-lice.html

```diff
@@ -25,102 +29,93 @@
   </div> <!-- end row -->
 
   <div class="condition-images">
-    
-      <figure>
-          <img src="/images/head-lice/01_head_lice.png" alt="lice" />
-          <figcaption>Head lice are small insects, up to 3 mm long.</figcaption>
-      </figure>
-      
-      <figure>
-          <img src="/images/head-lice/02_head_lice_zoom.png" alt="lice" />
-          <figcaption>Close up view of a head lice.</figcaption>
-      </figure>
-
-      <figure>
-          <img src="/images/head-lice/03-empty-eggs.jpg" alt="nits" />
-          <figcaption>The empty shells of the eggs after they hatch stay in the hair. These are called nits.</figcaption>
-      </figure>
-
+    <figure>
+      <img src="/images/head-lice/01_head_lice.png" alt="lice" />
+      <figcaption>Head lice are small insects, up to 3mm long.</figcaption>
+    </figure><!--
+    --><figure>
```

Comment:
Yes, it's one way to eradicate built-in element spacing when you're displaying `inline-block`. Have a quick look at https://css-tricks.com/fighting-the-space-between-inline-block-elements/